doc/man3/EVP_MD_fetch.pod

```diff
@@ -39,9 +39,10 @@ algorithm from the default provider.
 
 With explicit fetch an application uses the EVP_MD_fetch() function to obtain
 an algorithm implementation. An implementation with the given name and
-satisfying the search criteria specified in the B<properties> parameter will be
-looked for within the available providers and returned. See L<OSSL_PROVIDER(3)>
-for information about providers.
+satisfying the search criteria specified in the B<properties> parameter
+combined with the default search criteria will be looked for within the
```

Comment:
Perhaps a reference to EVP_set_default_properties would be good here.

Reply:
Done